packages/e2e-test-utils-playwright/src/page/switch-user.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { WP_ADMIN_USER, WP_USERNAME } from '../config';
+
+/**
+ * Switches the current user to the admin user (if the user
+ * running the test is not already the admin user).
+ *
+ * @this {import('./').PageUtils}
+ */
+export async function switchUserToAdmin() {
+	if ( ( await this.getCurrentUser() ) === WP_ADMIN_USER.username ) {
+		return;
+	}
+	await this.loginUser( WP_ADMIN_USER.username, WP_ADMIN_USER.password );
```

Comment:
I couldn't see a `loginUser` utility for playwright.

Reply:
We don't need this anymore